bots/pr/src/main/java/org/openjdk/skara/bots/pr/CommitCommentsWorkItem.java

```diff
@@ -67,12 +67,14 @@ private boolean isAncestor(ReadOnlyRepository repo, Hash ancestor, Hash descenda
             var hostedRepositoryPool = new HostedRepositoryPool(seedPath);
             var localRepoPath = scratchPath.resolve("pr").resolve("commit-comments").resolve(bot.repo().name());
             var localRepo = hostedRepositoryPool.materialize(bot.repo(), localRepoPath);
+            localRepo.fetchAllRemotes(false);
```

Comment:
Not sure this is needed? The materialize call should have done a fetchAll as well..